test/system/scroll_test.rb

```diff
@@ -27,7 +27,7 @@ class ScrollTest < ApplicationSystemTestCase
     refute_text "Developer number one"
 
     scroll_to_bottom_of_page
-    assert_text "Developer number one"
+    assert_text "Private information"
```

Comment:
Let's split this out into 2 tests: one for a subscriber (the existing test) and one for someone who sees the paywall (your updated test).